### ISSUE TITLE
Adding a <span> on icons text for css selecting?

### ISSUE THREAD

**Issue by Andi-Lo** [lukaskleinschmidt/kirby-sortable#14]
I got a suggestion to make to help the modules readability when using it with columns.

Currently i am using the modules on a page where i got a 2/3 - 1/3 column layout like this:

![image](https://cloud.githubusercontent.com/assets/7545568/20237376/70c4da84-a8d0-11e6-964d-5be75baf9654.png)

As you see there is really just not much space for displaying the modules name e.g. "infobox". Now my thoguth was, that I don't need the text for "edit" and "delete" because they need so much space. I would like to hide it. However the generated html does not have a selector for the text element and so you can't hide it with css. It is like this:

`<a href=""><i></i>edit</a>`
and I would like to have something like that:
`<a href=""><i></i><span class="maybe a class">edit</span></a>`

So my suggestion is to add in file `modules/styles/module.php` a surrounding <span> tag for the  text element.

```html
    <a class="modules-entry-button btn btn-with-icon modules-edit-button" href="<?php __($module->url('edit')); ?>">
      <?php i('pencil', 'left')?>
      <span><?php. _l('fields.structure.edit') ?></span>
    </a>
    <a data-modal class="modules-entry-button btn btn-with-icon modules-delete-button" href="<?php __($field->url('delete', array('uid' => $module->uid()))); ?>">
      <?php i('trash-o', 'left') ?>
      <span><?php _l('fields.structure.delete') ?></span>
    </a>
```

It would be great if the end result could be something like this:

![image](https://cloud.githubusercontent.com/assets/7545568/20237391/fa3b748a-a8d0-11e6-97b2-84ef6e9fd672.png)

Whats your thoughts on this one?

**Comment by lukaskleinschmidt** [comment]
I added an option for that.

``` yml
type: module
options:
  label: false
```

**Comment by Andi-Lo** [comment]
Oh thank you,

seems like I missed the update where you added that :) . Need to update the plugin.

Feel free to close

**Comment by lukaskleinschmidt** [comment]
well the update is 44minutes old 😃

**Comment by Andi-Lo** [comment]
:D oh okay so it didn't exist yet. My thought was that I did overlook it. Well done

By the way you can link commits to issue numbers on github:
http://stackoverflow.com/questions/1687262/link-to-the-issue-number-on-github-within-a-commit-message

Might be interesting for you. That way I would have instantly seen your related commit to this issue :)

**Comment by lukaskleinschmidt** [comment]
Thanks fot the tip. Will keep that in mind for the next time.

**Comment by Andi-Lo** [comment]
@lukaskleinschmidt  for me it seems there is an issue when I update to the latest version. I upgraded from: "version": "1.0.0"

![image](https://cloud.githubusercontent.com/assets/7545568/20428930/0e3f3a9c-ad8c-11e6-8128-57b18b55c2d4.png)

It does not recognize any modules anymore? I tried to flush any caching. When I create a new module it also does not get listed anymore.

EDIT: I recognized that you did a compatibility upgrade for the modules plugin in v 1.2.0. Upgrading the plugin solved the problem.

By the way. Your package.json should have a push in its version number (its still on 1.1.0)
